NightscoutUploadKit/Models/RemoteNotifications/BolusRemoteNotification.swift

```diff
@@ -0,0 +1,30 @@
+//
```

Comment:
These are the Codable models used for Nightscout. I'm including these with the NightscoutUploadKit which I feel is the "client" of Nightscout APIs. It could be argued this is more part of the "NightscoutService" domain as these are only exposed in the context of the plugin. Nevertheless there will be client things for Remote 2.0 later so I'm starting here to avoid fragmenting things.

Reply:
If they are things that any iOS app wanting to talk to Nightscout might use (upload, download), then it belongs in NightscoutKit. If it's specific to Loop, then it goes in NightscoutService.

Reply:
It sounds like all these Codable models should move to the NightscoutService as these are all Loop specific.

Do you see the role of NightscoutService as also serving _other apps_ like Caregiver, associated with Loop, that needs to make Loop-specific Nightscout api calls? 

If so, we would need to add something like a NightscoutLoopAPI to NightscoutService that handles those network things, similarly to the NigthscoutAPI.  I ask as the the notifications endpoint in NightscoutKit is currently doing some Loop domain things so probably doesn't belong there.

Reply:
Nightscout is being used as a "dumb pipe" as you mentioned. I had considered a more domain-aware API as you mentioned but didn't want to get ahead of myself. I can work towards moving that direction when we get there.

On that matter, the NS reviewer sulkaharo had suggested on my NS [Draft PR](https://github.com/nightscout/cgm-remote-monitor/pull/7791) that this remote commands feature be coordinated with the wider community to potentially make it cross-compatible with other implementations (Like AndroidAPS). Your comments though also suggest a move in that direction.